Doc/library/asyncio-stream.rst

```diff
@@ -206,12 +206,21 @@ StreamReader
 
    .. coroutinemethod:: read(n=-1)
 
-      Read up to *n* bytes.  If *n* is not provided, or set to ``-1``,
-      read until EOF and return all read bytes.
+      Read up to *n* bytes from the stream.
 
+      If *n* is not provided, or set to ``-1``,
+      read until EOF, then return all read :class:`bytes`.
```

Comment:
As a beginning reader, before having read @jgosmann 's [detailed explaination](https://github.com/python/cpython/pull/101807#discussion_r1108985811) and @gvanrossum 's summary, I was evidently confused here about what "until EOF" specifically meant in this context. It therefore might warrant some more clarity here, but maybe its just my lack of subject matter expertise compared with the target audience.